contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -383,7 +383,7 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
             return
                 RewardsMath.calcFlatRewards(p.totalRewards, currTimeElapsed - prevTimeElapsed, p.endTime - p.startTime);
         } else {
-            // if (p.distributionType == EXPONENTIAL_DECAY_DISTRIBUTION)
+            // if (p.distributionType == EXP_DECAY_DISTRIBUTION)
```

Comment:
Do we really need this comment?

Reply:
IMO, it covers for the inherent lack of a (self-explanatory) switch/case statement.

Reply:
It looks like an accidental copy-paste from the contracts. Can you rephrase it?

Reply:
done